clase9/auto2.ts

```diff
@@ -0,0 +1,33 @@
+import {Mainauto} from "./autopadre"
+class Auto {
+    private marca: string;
+    private año:number;
+   private precio: number;
+  
+constructor(){
+this.marca=this.marca;
+this.año=this.año;
+this.precio=this.precio;
```

Comment:
this.marca = this.marca no retorna ningun valor. Asignale un valor por default o por parametro del constructor
Ademas usa el super() en el constructor para los atributos que herede del padre, asi te ahorras de crear devuelta las variables marca y precio ya que estan heredadas.